content/applications/hr/employees/new_employee.rst

```diff
@@ -183,42 +195,48 @@ this section may be critical for the company's payroll department. In order to p
-box, or typing in the information.
+Here, the employee's :guilabel:`Private Contact`, :guilabel:`Education`, :guilabel:`Work Permit`,
+:guilabel:`Family Status`, :guilabel:`Emergency Contact`, and :guilabel:`Citizenship` information is
+entered. Fields are entered either using a drop-down menu, clicking a check box, or typing in the
+information.
 
-- :guilabel:`Private Contact`: Enter the :guilabel:`Address` for the employee. The selection can be
-  made with the drop-down menu. If the information is not available, type in the name for the new
-  address. To edit the new address, click the :guilabel:`External Link` button to open the address
-  form. On the address form, enter the necessary details, then click :guilabel:`Save`. Some other
+- :guilabel:`Private Contact`: Enter the personal :guilabel:`Address` for the employee. The
+  selection can be made with the drop-down menu. If the information is not available, type in the
+  name for the new address. To edit the new address, click the :guilabel:`Internal Link` button to
+  open the address form. On the address form, enter or edit the necessary details. Some other
   information in the :guilabel:`Private Contact` section may auto-populate, if the address is
-  already listed in the drop-down menu.
-- :guilabel:`Marital Status`: Select either :guilabel:`Single`, :guilabel:`Married`,
-  :guilabel:`Legal Cohabitant`, :guilabel:`Widower`, or :guilabel:`Divorced` from the drop-down
-  menu.
-- :guilabel:`Emergency Contact`: Type in the name and phone number of the employee's emergency
-  contact.
+  already listed in the drop-down menu. Enter the employee's :guilabel:`Bank Account Number` using
+  the drop-down menu. If the bank is not already configured (the typical situation when creating a
+  new employee) enter the bank account number, and click :guilabel:`Create and edit..`. A
+  :guilabel:`Create Bank Account Number` for appears. Fill in the information, then click
+  :guilabel:`Save & Close`. Then, select the employee's preferred :guilabel:`Language` from
+  the drop-down menu. Next, enter the :guilabel:`Home-Work Distance` in the field. This field is
+  only necessary if the employee is receiving any type of commuter benefits. Last, enter the
+  employee's license plate information in the :guilabel:`Private Car Plate` field.
 - :guilabel:`Education`: Select the highest level of education completed by the employee from the
   :guilabel:`Certificate Level` drop-down menu. Options include :guilabel:`Graduate`,
   :guilabel:`Bachelor`, :guilabel:`Master`, :guilabel:`Doctor`, or :guilabel:`Other`. Type in the
   :guilabel:`Field of Study`, and the name of the :guilabel:`School` in the respective fields.
+- :guilabel:`Work Permit`: If the employee has a work permit, enter the information in this section.
+  Type in the :guilabel:`Visa No` and/or :guilabel:`Work Permit No` in the corresponding fields.
```

Comment:
Does "No" mean "Number" in this case? Would be great to clarify this.

Sidenote, if it does mean "number," these should really have a period after "No." Maybe something you could submit to the feedback team.